common/lib/xmodule/xmodule/tests/test_html_module.py

```diff
@@ -244,3 +277,4 @@ def test_updates_order(self):
             template_name,
             expected_context
         )
+
```

Comment:
@bradmerlin nit: unnecessary newline - miht be good to remove to reduce the diff.